index.md

```diff
@@ -49,6 +49,8 @@ for the tutorial material
 > tasks on a remote machine (by `ssh` connection)
 > and in a cluster (high performance computing) environment.
 >
+> You should also be familiar with the [subset of YAML][yaml-for-cwl] used in CWL.
```

Comment:
This seems like an unhelpful statement to new learners. What does "familiar with" mean? Just reading it?

Having the link to the yaml-for-cwl guide is good, but I don't expect learners to read that at first; more of a reference to come back to later after then have done these lessons.

```suggestion
> CWL is based upon YAML. At any time, if you find yourself being confused by the YAML syntax, considering reviewing this guide on the [subset of YAML][yaml-for-cwl] used in CWL.
```

Thoughts?

Reply:
Yep, good suggestion!